autoload/leaderf/python/leaderf/fileExpl.py

```diff
@@ -725,6 +732,13 @@ def _afterEnter(self):
         lfCmd("autocmd VimLeavePre * call leaderf#File#cleanup()")
         lfCmd("augroup END")
 
+        winid = self._getInstance().getPopupWinId() if self._getInstance().getWinPos() == 'popup' else None
```

Comment:
put this line under the if statement below.

Reply:
I fixed.